docs/ref/views.txt

```diff
@@ -61,7 +61,12 @@ these with your own custom views, see :ref:`customizing-error-views`.
 The 404 (page not found) view
 -----------------------------
 
-.. function:: defaults.page_not_found(request, template_name='404.html')
+.. function:: defaults.page_not_found(request, exception, template_name='404.html')
+
+.. versionchanged:: 1.9
+
+    The signature of ``page_not_found()`` changed in Django 1.9. The function
```

Comment:
Since this renders with "Changed in Django 1.9:" as a prefix, I think "change in Django 1.9" is a bit redundant.